src/BankConsole/CompareBankAccounts.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using BankingSystem.Bank;
+
+namespace BankingSystem
+{
+    public class CompareBankAccounts
```

Comment:
why do we need this? if you want to test the compare should not be it a TEST?

Reply:
This class is an exercise, but I also think that is more efficient test the functionality, I will change it.

Reply:
You say you will change but I don´t see any change

Reply:
I already deleted it ;)